app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledInlineTriggerRow.tsx

```diff
@@ -0,0 +1,131 @@
+
+const TRIGGER_PREFIX = 'features.PipelineTriggers';
+
+interface IEnabledInlineTriggerRowProps {
+  isExpanded: boolean;
+  trigger: any;
+  groupSchedule: any;
+  loading: boolean;
+  info: any;
+  pipelineName: string;
+  disableError: string;
+}
+
+export default function EnabledInlineTriggerRow(props: IEnabledInlineTriggerRowProps) {
+  const statePipelineName = PipelineTriggersStore.getState().triggers.pipelineName;
+
+  const renderLoading = () => {
+    return (
+      <div className="text-center">
+        <LoadingSVG />
+      </div>
+    );
+  };
+
+  const renderContent = () => {
+    const { trigger, groupSchedule, info, pipelineName } = props;
+
+    const namespace = NamespaceStore.getState().selectedNamespace;
+    const triggeredPipelineInfo = {
+      id: pipelineName,
+      namespace,
+    };
+
+    const triggeringPipelineInfo = {
+      id: trigger.programId.application,
+      namespace: trigger.programId.namespace,
+    };
+
+    const events = trigger.programStatuses;
+    const completed = events.indexOf('COMPLETED') > -1;
+    const killed = events.indexOf('KILLED') > -1;
+    const failed = events.indexOf('FAILED') > -1;
+
+    return (
+      <div className="row-content">
+        <div className="pipeline-description">
+          <strong>{T.translate(`${TRIGGER_PREFIX}.description`)}: </strong>
+          <span>{info && info.description}</span>
+          <a
+            href={`/pipelines/ns/${triggeringPipelineInfo.namespace}/view/${triggeringPipelineInfo.id}`}
+          >
+            {T.translate(`${TRIGGER_PREFIX}.viewPipeline`)}
+          </a>
+        </div>
+
+        <div className="helper-text">
+          {T.translate(`${TRIGGER_PREFIX}.helperText`, { pipelineName: statePipelineName })}
+        </div>
+
+        <div className="events-list">
+          <div className="checkbox-item">
+            <IconSVG name={completed ? 'icon-check-square' : 'icon-square-o'} />
+            <span>{T.translate(`${TRIGGER_PREFIX}.Events.COMPLETED`)}</span>
+          </div>
+
+          <div className="checkbox-item">
+            <IconSVG name={killed ? 'icon-check-square' : 'icon-square-o'} />
+            <span>{T.translate(`${TRIGGER_PREFIX}.Events.KILLED`)}</span>
+          </div>
+
+          <div className="checkbox-item">
+            <IconSVG name={failed ? 'icon-check-square' : 'icon-square-o'} />
+            <span>{T.translate(`${TRIGGER_PREFIX}.Events.FAILED`)}</span>
+          </div>
+        </div>
+
+        {props.disableError && <div className="text-danger error">{props.disableError}</div>}
+
+        <div className="action-buttons-container">
+          <PayloadConfigModal
+            triggeringPipelineInfo={triggeringPipelineInfo}
+            triggeredPipelineInfo={triggeredPipelineInfo}
+            scheduleInfo={groupSchedule}
+            disabled={true}
+          />
+        </div>
+      </div>
+    );
+  };
+
+  return !props.isExpanded ? (
+    <div
```

Comment:
if you have to use divs instead of buttons please check this out to make it semi accessible https://benfrain.com/converting-divs-into-accessible-pseudo-buttons/